### PR TITLE
WIP: Introduction text

### PR DESCRIPTION
**Pull request by icweaver**
Added some words =]

### CODE REVIEW

**Comment by giordano** [comment]
You'll need to add the file to [the list of pages](https://github.com/JuliaAstro/Transits.jl/blob/027b0618a191444d2f50b161be92c36936f33b19/docs/make.jl#L20-L25) of the documentation :wink:

**Review by giordano — COMMENTED** [review]
Looks overall good to me, but I'd add references for the images as well :slightly_smiling_face:

**Comment by mileslucas** [comment]
This is great so far! Something you might want to sneak in is the answer to the question "what is limb darkening?", since it is prominent term in the description of the package.

Thanks again- when you're happy with your revisions I foresee a swift merge.

**Comment by icweaver** [comment]
Thanks all for the feedback, the next iteration should be up!

File: docs/make.jl [hunk 1]
```diff
@@ -20,6 +20,7 @@ makedocs(;
     pages=[
         "Home" => "index.md",
         "Getting Started" => "gettingstarted.md",
+        "Introduction" => "introduction.md"
```
Comment (giordano) [inline docs/make.jl:23]:
```suggestion
        "Introduction" => "introduction.md",
```

Comment (icweaver) [reply docs/make.jl]: Thanks!

**Comment by icweaver** [comment]
Ok, looks like things should be rebased and ready to merge!

**Review by mileslucas — APPROVED** [review]

**Pull request merged**